tests/tasti_test/tasti_test.py

```diff
@@ -143,3 +151,9 @@ def test(
     weaviate_auth = WeaviateAuth(url, api_key=api_key)
     test('Weaviate', VectorDatabaseType.WEAVIATE.value, data_size=200,
          embedding_dim=128, nb_buckets=50, weaviate_auth=weaviate_auth)
+    
+    logger.info(f'Running Marqo vector database')
+    url = 'http://0.0.0.0:8882'
```

Comment:
Have you successfully run the test? I met following error:
marqo.errors.BackendCommunicationError: MarqoWebError: BackendCommunicationError Error message: Error communicating with Marqo: HTTPConnectionPool(host='0.0.0.0', port=8882): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd682c16280>: Failed to establish a new connection: [Errno 61] Connection refused'))
status_code: 500, type: internal, code: backend_communication_error, link: 
Could you tell me how to solve it?

Reply:
@ttt-77 I am able to run the test on my `ubuntu` system. Which machine are you using? Also, you can check if the instance is started by making a `curl http://localhost:8882` request, it should give `{"message":"Welcome to Marqo","version":"1.5.0"}%`. 
If you are using M series Mac, please [refer here](https://github.com/marqo-ai/marqo#m-series-mac-users) or try below commands:
1. Start opensearch
```
docker rm -f marqo-os; docker run -p 9200:9200 -p 9600:9600 -e "discovery.type=single-node" marqoai/marqo-os:0.0.3-arm
```
2. Launch marqo
```
docker rm -f marqo; docker run --name marqo --privileged \
    -p 8882:8882 --add-host host.docker.internal:host-gateway \
    -e "OPENSEARCH_URL=https://localhost:9200" \
    marqoai/marqo:latest
```

You can also add this parameter - `MARQO_MODELS_TO_PRELOAD='[]'` to not use any default models, because we are anyway providing the embeddings, it will save some RAM usage.

Reply:
Have you met this error?
    raise MarqoWebError(message=response_msg, code=code, error_type=error_type,
marqo.errors.MarqoWebError: MarqoWebError: MarqoWebError Error message: {"detail":[{"loc":["body"],"msg":"value is not a valid list","type":"type_error.list"}]}
status_code: 422, type: unhandled_error_type, code: unhandled_error, link:

Reply:
No, I am not getting this, can you provide the stack trace, or which lines lead to this error?  so that I can debug

Reply:
![Uploading Screenshot 2023-12-19 at 17.37.00.png…]()

Reply:
Image not visible to me, the link is to this PR itself, not the image.

Reply:
<img width="1462" alt="Screenshot 2023-12-19 at 17 44 46" src="https://github.com/ddkang/aidb/assets/43330557/045db26a-0f27-48dd-924e-411a7b9fabf6">

Reply:
Its an issue with the marqo version, can you do this - `docker pull marqoai/marqo:latest`, this would fetch you marqo's latest version. Can you try with this once?